agent/fw_slim.c

```diff
@@ -120,7 +122,10 @@ NR_PHP_WRAPPER(nr_slim_application_construct) {
   version = nr_php_get_object_constant(this_var, "VERSION");
   
   // Add php package to transaction
-  nr_txn_add_php_package(NRPRG(txn), "slim/slim", version);
+  nr_txn_add_php_package(NRPRG(txn), PHP_PACKAGE_NAME, version);
```

Comment:
is this always added, even if vm is disabled?

Reply:
Fixed: [647d424](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/647d424ffb824368d14f803f6812df43a75be043)